game/game.py

```diff
@@ -386,25 +386,33 @@ def set_bullseye(self) -> None:
         self.blue_bullseye = Bullseye(enemy_cp.position)
         self.red_bullseye = Bullseye(player_cp.position)
 
-    def initialize_turn(self) -> None:
-        self.events = []
-        self._generate_events()
+    def initialize_turn(self, is_reinit: bool = False, for_player: bool = True) -> None:
```

Comment:
I don't think `for_player` is the right name? Both red and blue actions are taken when this is true. Based on the behavior I think I'm reading, I think it's most clear if inverted and named `opfor_only`?

Reply:
i changed the whole thing to `for_coalition: List[str] = ["Blue","Red"]` so that you can set for which coalition you want it, or both